subsys/zigbee/osif/zb_nrf_platform.c

```diff
@@ -545,6 +546,11 @@ void zb_reset(zb_uint8_t param)
 	nrf_power_gpregret_set(NRF_POWER, reas);
 #endif /* CONFIG_SOC_NRF5340_CPUAPP */
 
+	/* Power on unused sections of RAM to allow MCUboot to use it. */
```

Comment:
What do you think about adding dependency to MCUBoot here?

Reply:
It is not needed. Generally speaking - any bootloader expects to have a full RAM available on cold boot.
If necessary, I may change "MCUboot" to "bootloader", but I do not see the need for that.

Reply:
Agree, no need to be specific about which bootloader to use.